ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -66,6 +87,20 @@ class PropertySpec: QuickSpec {
 				expect(sentValue) == subsequentPropertyValue
 			}
 
+			it("should yield a signal that sends the all the changes") {
+				let mutableProperty = MutableProperty(initialPropertyValue)
+
+				var sentValue: String?
+
+				mutableProperty.signal.observeNext { value in
+					sentValue = value
+				}
+
+				expect(sentValue).to(beNil())
+				mutableProperty.value = subsequentPropertyValue
+				expect(sentValue) == subsequentPropertyValue
```

Comment:
It might be worth adding another modification to verify that it keeps emitting changes.
Also, can you add a separate test to verify that it also emits values when the property is set **even though the value didn't change**?